libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -89,6 +99,9 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
   }
 
   @Input() set minDate(value: Date) {
+    if (value && this.activeMonth.toDate() < value) {
+      this.setActiveMonth(value);
+    }
```

Comment:
Is this unit tested?

Reply:
Yes, but actually no. It has a test under describe('active month', [...]), however not all return paths are covered in the test. Current test only verifies that the view actually changes whenever `minDate` or `maxDate` changes respectively to a date later or earlier than `todayDate`.